src/sentry/api/endpoints/organization_events_trace.py

```diff
@@ -1013,11 +1053,6 @@ def get(self, request: HttpRequest, organization: Organization, trace_id: str) -
         if event_id and not is_event_id(event_id):
             return Response({"detail": INVALID_ID_DETAILS.format("Event ID")}, status=400)
 
-        tracing_without_performance_enabled = features.has(
-            "organizations:performance-tracing-without-performance",
-            organization,
-            actor=request.user,
-        )
```

Comment:
Is this flag fully GA'ed now? Are there environments where this can't be turned on like single tenant or self hosted?

Reply:
Fully GA'ed, the frontend already assumes this flag is on since it changes the response 😬